backends/postgres.go

```diff
@@ -98,6 +98,14 @@ func NewPostgres(authOpts map[string]string, logLevel log.Level, hasher hashing.
 	}
 
 	if sslmode, ok := authOpts["pg_sslmode"]; ok {
+		switch sslmode {
+		case "verify-full":
```

Comment:
Cases may be combined as comma separated values if I recall correctly.

Reply:
Fixed